src/plugins/vis_type_metric/public/index.scss

```diff
@@ -7,4 +7,4 @@
 //   mtrChart__legend--small
 //   mtrChart__legend-isLoading
 
-@import './metric_vis';
```

Comment:
The `styling_constants` import at the top of this file is no longer necessary.

Reply:
thx, I've removed it!